.github/workflows/ci-master.yml

```diff
@@ -38,38 +38,28 @@ jobs:
       run: |
         sudo apt update
         sudo apt install -y ccache g++-mingw-w64-x86-64 mingw-w64-x86-64-dev
-    - name: Update gcc mingw
-      run: sudo update-alternatives --set x86_64-w64-mingw32-gcc /usr/bin/x86_64-w64-mingw32-gcc-posix
-    - name: Update g++ mingw
-      run: sudo update-alternatives --set x86_64-w64-mingw32-g++ /usr/bin/x86_64-w64-mingw32-g++-posix
+    - name: Switch MinGW GCC and G++ to POSIX Threading
+      run: |
+        sudo update-alternatives --set x86_64-w64-mingw32-gcc /usr/bin/x86_64-w64-mingw32-gcc-posix
+        sudo update-alternatives --set x86_64-w64-mingw32-g++ /usr/bin/x86_64-w64-mingw32-g++-posix
     - name: Build dependencies
-      run: make -C depends -j2 HOST=x86_64-w64-mingw32
+      run: make -C depends -j$(nproc) HOST=x86_64-w64-mingw32
     - name: Build Zcoin
       run: |
         ./autogen.sh
         ./configure --disable-jni --enable-exodus --prefix=$(pwd)/depends/x86_64-w64-mingw32
-        make -j2
+        make -j$(nproc)
 
   target-macos:
     name: Target macOS
-    runs-on: ubuntu-latest
+    runs-on: macos-latest
     steps:
     - name: Checkout
       uses: actions/checkout@v1
     - name: Install dependencies
-      run: |
-        sudo apt update
-        sudo apt-get install -y ccache libcap-dev
-    - name: Prepare directory for SDKs
-      run: mkdir -p depends/SDKs
-    - name: Download SDK
-      run: curl -L -o /tmp/MacOSX10.11.sdk.tar.gz https://bitcoincore.org/depends-sources/sdks/MacOSX10.11.sdk.tar.gz
-    - name: Extract SDK
-      run: tar -C depends/SDKs -xf /tmp/MacOSX10.11.sdk.tar.gz
-    - name: Build dependencies
-      run: make -C depends -j2 HOST=x86_64-apple-darwin11
+      run: brew install automake berkeley-db4 libtool boost miniupnpc pkg-config python qt libevent qrencode ccache
     - name: Build Zcoin
       run: |
         ./autogen.sh
-        ./configure --enable-exodus --prefix=$(pwd)/depends/x86_64-apple-darwin11
-        make -j2
\ No newline at end of file
+        ./configure --enable-exodus
+        make -j$(nproc)
```

Comment:
Seem like `$(nproc)` is not working on macOS.

Reply:
correct. you can access `nproc` on MacOS with `sysctl -n hw.physicalcpu`.